xml/ReportingOrganisation.xml

```diff
@@ -13048,7 +13048,7 @@
             <codeforiati:hq-country-or-region>United States</codeforiati:hq-country-or-region>
             <codeforiati:registry-identifier>mdif</codeforiati:registry-identifier>
         </codelist-item>
-        <codelist-item status="active">
+        <codelist-item status="withdrawn">
```

Comment:
Hmm I am not sure why this would be withdrawn... it is still available here:
https://iatiregistry.org/publisher/mdif

Reply:
Oh yeah – confirmed (via the CSV)…

That’s weird. I’ll re-run the github action, I suspect it was a temporary glitch.

Reply:
Oh wait hang on – that’s not the correct publisher. It’s this one:
https://iatiregistry.org/publisher/fd-es

Publishers without any datasets are not listed in the registry’s CSV of all publishers, so that’s why it’s marked as withdrawn here.

Reply:
Ah OK, thanks! Sorry for my mistake

Reply:
No prob! The diff is deceptive. I was also fooled (but deleted my original reply!)